src/layouts/DefaultAppBar.js

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Stack from "@mui/material/Stack";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import IconButton from "@mui/material/IconButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import { ThemeProvider, createTheme } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+
+function appBarLabel(label) {
```

Comment:
Can you update the name to `AppBarLabel` to be consistent with other company? Besides that the file name is DefaultAppBar.js, but non of the function name is matching. Can you update it?

Reply:
done